src/ctia/store.clj

```diff
@@ -28,7 +28,7 @@
   (create-events [this new-events]))
 
 (defprotocol IQueryStringSearchableStore
-  (query-string-search [this search-query ident params])
+  (query-string-search [this args #_#_:- QueryStringSearchArgs])
```

Comment:
```suggestion
  (query-string-search [this args :- QueryStringSearchArgs])
```

Reply:
Turns out `s/defprotocol` doesn't exist in our schema version. I'll leave a note, but I don't want to mess with our deps in this PR.